mitodl/odl-video-service
# OVS Sandwich menu content

Fixes #165 

#### What's this PR do?
Adds the following to the sandwich menu:
- The user's email (if it exists) or username (should be the same w/Touchstone)
- A list of collections the user has admin access to
- A logout button

#### How should this be manually tested?
Go to any video detail page and click the sandwich menu button at top left to open it.  The menu should display all the above.
